### DISCUSSION TITLE
RFC: RFC Process

### DISCUSSION THREAD

**Discussion by cor** [cwwg/cwwg#6] [RFCs]
This RFC describes the RFC process, introduced in cwwg/cwwg#8

**Comment by aeryz** [comment]
I think the process is nice, descriptive and clean. One thing is for the discussions, we can have labels to show the status of an RFC, so in the future, people can easily categorize RFC's. I am thinking of having labels:
1. `RFC-discussion`: RFC's that are being discussed.
2. `RFC-active`: RFC's that are active, which means they are approved and will be implemented in the future.
3. `RFC-done`: RFC's that were active and are implemented.
4. `RFC-rejected`: RFC's that are not active and not going to be implemented.

**Reply by Kiruse** [reply]
so an RFC that doesn't need to be implemented would go straight to `RFC-done`?

**Reply by aeryz** [reply]
Yeap. This RFC is a good example of that.

**Comment by Kiruse** [comment]
very orderly & structured. I'm in favor. however, additionally to the two core members required for acceptance, I'd like to add a minimum time box, say 2 weeks, that an RFC has to remain in discussions in order to warrant ample consideration of the RFC. perhaps a longer time box depending on how much time the core team can allocate toward the WG

**Reply by aeryz** [reply]
I am up for the idea! Because two people can instantly approve but other people might not have the time to even read the RFC yet.  
I also think 2 weeks is sufficient because longer might be counterproductive. Normally in a core project, like cosmwasm itself, 2 weeks would probably be too short but since at first, we will be focusing on tooling and docs, ease of development process should be our priority.
